src/main/java/com/mathworks/ci/RunMatlabBuildBuilder.java

```diff
@@ -151,8 +161,10 @@ private void createMatlabScriptByName(FilePath uniqueTmpFolderPath, String uniqu
             cmd += " " + tasks;
         }
 
+        String buildScript = "buildRunner('"+tasks+"')";
+
         final String matlabCommandFileContent =
-                "cd '" + workspace.getRemote().replaceAll("'", "''") + "';\n" + cmd;
+                "addpath(pwd);cd '" + workspace.getRemote().replaceAll("'", "''") + "';\n" + buildScript;
 
         // Display the commands on console output for users reference
```

Comment:
This isn't true anymore. We're not just calling `buildtool mytask`. We're calling a script that's very different.

Reply:
Done. removed the buildtool + task